doc/md/privacy.md

```diff
@@ -469,51 +472,51 @@ func DenyMismatchedTenants() privacy.MutationRule {
 		}
 		// Query the tenant-id of all users. Expect to have exact 1 result,
 		// and it matches the tenant-id of the group above.
-		uid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
+		userTid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
 		if err != nil {
-			return privacy.Denyf("querying the tenant-id %w", err)
+			return privacy.Denyf("querying the tenant-id %v", err)
```

Comment:
Are we sure we want `%v` here? I'm not sure of the internals, but ideally the error is embedded with `%w` so it can be unwrapped.

Reply:
You are correct, we do want `%w`. This is a mistake from a previous PR - #1275 it changed the `.md` file but not the example file `rule.go`, so when I copied it, it was overwritten. Fixing it.